File: cartservice/src/main/java/com/eshopping/cartservice/model/Transaction.java
package com.eshopping.cartservice.model;

public class Transaction {
	
	private String id;
	private String sender;
	private String receiver;
	private double amount;
	
	
	public String getId() {
		return id;
	}
	public void setId(String id) {
		this.id = id;
	}
	
	public String getSender() {
		return sender;
	}
	
	public void setSender(String sender) {
		this.sender = sender;
	}
	
	public String getReceiver() {
		return receiver;
	}
	
	public void setReceiver(String receiver) {
		this.receiver = receiver;
	}
	
	public double getAmount() {
		return amount;
	}
	
	public void setAmount(double amount) {
		this.amount = amount;
	}
	
	

}

File: ECommerce App/EShoppingZone/src/app/Models/transaction.ts
export class Transaction {
    id:string;
    sender:string;
    receiver:string;
    amount:number;
    constructor(id:string,sender:string,receiver:string,amount:number){
        this.id=id;
        this.sender=sender;
        this.receiver=receiver;
        this.amount=amount;
    }
}
